src/auth/auth.ts

```diff
@@ -110,7 +117,9 @@ export const signOut = (cause: SignOutCause = 'unspecified'): void => {
   azureCookieReadyStore.reset();
   getSessionStorage().clear();
 
-  revokeTokens();
+  await cookieProvider.invalidateCookies().catch(_.noop);
```

Comment:
Not keen on ignoring _all_ errors here. I think that risks providing false confidence in this function.

The only thing we actually want to _ignore_ is 401 responses caused by expired tokens, because those mean the cookie is already invalid.

Reply:
Is there a way to send an anonymous metric to Mixpanel here? I'd love to be able to track that an error did occur, even if we can't identify who encountered it.

Reply:
Yes, sending metric events works fine when the user is signed out. It'll even be identifiable, since the "anonymous ID" used for metrics when signed out is linked to a user ID when they sign in.

Reply:
Hmm... I agree that it would be nice to be more specific but there are three reasons that led me to do it this way

1) In the auth function, any errors could result in sign out breaking. Without a good set of contract tests, I think it's good to be as defensive as possible. This is mitigated by the fact that we only ignore all errors in the signOut function, not at the layer of the provider, so other sources that invalidate cookies will still fail
2) This is already how we do it for existing calls in signOut, particularly metrics. The `withErrorIgnoring` wrapper is used, and its used a good bit of other places. 
3) And the biggest, the approach of only failing on a 401 is not easily testable without an e2e test ☹️ . We would need an e2e that calls these endpoints with an invalid token to ensure the error handling does the proper thing without casting a broad net.